Docs/ArchitectureDecisions/003_region_as923_implementation.md

```diff
@@ -0,0 +1,33 @@
+# 001. Region AS923 implementation
+
+Milestone / Epic: [#412](https://github.com/Azure/iotedge-lorawan-starterkit/issues/412)
+
+Authors: Maggie Salak
+
+## Overview / Problem Statement
+
+The specification of region AS923 defines a `AS923_FREQ_OFFSET` parameter which is used to accommodate different country-specific sub-bands across the 915 - 928 MHz band. The parameter can have one of four different values depending on a country. The corresponding frequency offset in Hz is `AS923_FREQ_OFFSET_HZ = 100 x AS923_FREQ_OFFSET`. The value of the parameter is needed for the purpose of calculating RX2 window frequencies for AS923 region. The parameter is not required for calculating RX1 receive window as it simply uses the same channel as the preceding uplink.
+
+This document summarizes decisions taken for the purpose of implementing support for region AS923.
+
+## In-Scope
+
+- Support for all countries using frequency plan AS923
+- Calculation of RX1 downstream frequencies and data rates
+- Calculation of RX2 receive window
+
+## Out-of-scope
+
+- MAC commands - support will be added later on as part of [#414](https://github.com/Azure/iotedge-lorawan-starterkit/issues/414)
```

Comment:
Probably you can mention dwell time here (and create an associated wotrk item, we need to get to the bottom of this)

Reply:
Done, also mentioned FOpts parameter which is tracked in another story.